### ISSUE TITLE
Multipath transport parameter should negotiate which version of the draft we are using

### ISSUE THREAD

**Issue by qdeconinck** [quicwg/multipath#139]
Now that we are starting seeing several multipath implementations and interoperability works between them, it could be useful to keep in mind to negotiate different transport parameter values when raising new draft versions, especially if we are going to add new frames and/or change their type ids.

**Comment by obonaventure** [comment]
You would change the transport parameter so that one bit includes the multipath support and the other bits a version number or add another transport parameter which is informal and provides the draft version ?

**Comment by mirjak** [comment]
I think as long as people don't start deploying it, I would prefer to only have one number and not separate draft numbers.

**Comment by LPardue** [comment]
I think that ship might have already sailed. Best to pick a versioning approach now and just roll with it: I see no benefit in delaying

**Comment by LPardue** [comment]
+1 Quentin

**Comment by huitema** [comment]
+1 Quentin. One parameter per version is much cleaner.

**Comment by mirjak** [comment]
Okay, but I don't think the next revision will need a new value as I believe we so far only merged clarification, right? Or do you already want a new value because the old one is already not compatible with even older version?

Should we add a sentence that non compatible version in future will use a new value? Or is there no need to explicitly say that in the draft and make sure we do it next time?

**Comment by qdeconinck** [comment]
For this version, if until the draft cut-off we do not merge anything breaking and only included clarification/editorial changes, I would be ok. But starting from the next version, I would expect different TP values.

I don't actually expect any additional text, but that was rather an issue I raised when we kept the same TP value while PATH_STATUS was present in one version but not the other. Of course, we can add some text in a "to be removed before being an RFC" section to specify this, but this is not required for me.
